src/jit/codegencommon.cpp

```diff
@@ -10162,11 +10162,14 @@ void CodeGen::genCaptureFuncletPrologEpilogInfo()
 #endif // DEBUG
 
         assert(PSP_slot_CallerSP_offset < 0);
-        assert(compiler->lvaPSPSym != BAD_VAR_NUM);
-        assert(PSP_slot_CallerSP_offset == compiler->lvaGetCallerSPRelativeOffset(compiler->lvaPSPSym)); // same offset
-                                                                                                         // used in main
-                                                                                                         // function and
-                                                                                                         // funclet!
+        if (!compiler->IsTargetAbi(CORINFO_CORERT_ABI)) // PSPSym is not used by the CoreRT ABI
```

Comment:
I guess PSP_slot_CallerSP_offset is just garbage in CoreRT? And the only interesting thing here is that lvaPSPSym is undefined for CoreRT?